k6ReportPortal.js

```diff
@@ -37,92 +42,132 @@ export function finishLaunch(launchId,reporterOptions) {
  * @returns 
  */
 function getHeader(token) {
-    var header = {
+    return {
         headers: {
-            "Content-Type": "application/json",
+            'Content-Type': 'application/json',
             authorization: `Bearer ${token}`,
         },
     };
-    return header;
-}
-
+};
 
 /**
  * 
  */
 export default class RpClient {
-
-    constructor(launchId,reporterOptions){
+    constructor(launchId, reporterOptions) {
         this.launchId = launchId
         this.reportPortalUri = `${reporterOptions.endpoint}/${reporterOptions.project}`
         this.token = reporterOptions.token
+    }
 
+    addLogToBatch(jsonObject, message, level = 'error') {
+        let newObject = jsonObject;
+        newObject.push({
+            itemUuid: uuidv4(),
```

Comment:
random uuid is not correct. This id must either test uuid or test step uuid.

Reply:
According to [dev guide](https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/reporting.md#batch-save-logs), it is supposed to be a UUID of the created log. Since you create those logs yourself, generating and ID yourself should be okay.

Reply:
Apologies. Disregard my last reply, you are correct. Item ID is not required at all, it is generated by reportPortal and is returned, not passed by agent. Fix on the way.